draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -413,24 +418,23 @@ field is not used and its size is 0.
 ### Uplink fragmentation: From device to SCHC gateway
 
 In that case the device is the fragmentation transmitter, and the SCHC gateway
-the fragmentation receiver.
-A single fragmentation rule is defined.
+the fragmentation receiver. A single fragmentation rule is defined.
 SCHC F/R MUST concatenate FPort and LoRaWAN payload to retrieve the SCHC
-fragment as per {{Fig-lorawan-schc-payload}}.
+fragment as per {{lorawan-schc-payload}}.
 
-* SCHC header is two bytes, including FPort byte.
-* **RuleID**: Recommended size is 8 bits in SCHC header.
-* **SCHC fragmentation reliability mode**: `ACK-on-Error`
-* **DTag**: Size is 0 bit, not used
-* **FCN**: The FCN field is encoded on N = 6 bits, so WINDOW_SIZE = 64 tiles
+* SCHC header is two bytes (the FPort byte + 1 additional byte).
```

Comment:
Should we specify `Minimum SCHC header is two bytes` ? As RuleId can be more than 1 byte

Reply:
I agree that maybe we are too restrictive in this case. Please see it my new change resolves your comment and resolve it if it is the case :slightly_smiling_face: